### ISSUE TITLE
Lich sinister gaze bug

### ISSUE THREAD

**Issue by Luojunya1991** [ValveSoftware/Dota2-Gameplay#13312]
### Ability name

Sinister gaze

### Description

Sinister gaze’s skill description states that the skill doesn’t  pierce spell immunity, which means it should not work on BKB units. 

However, the skill can interrupt skills such as Spirit breakers charge and Primal Beasts onslaught when they are under the effect of BKB. 

I have tested several scenarios. 
1. Hero cast BKB, lich cast sinister gaze on the hero,  sinister gaze has no effect, the hero can move and cast freely. 
2. Hero cast BKB, then charges( spirit breaker) or onslaught( primal beast), lich cast sinister gaze, the ability is interrupted. But it only interrupts the skill, it doesn’t cc the hero. For onslaught, it doesn’t matter if primal beast is still charging the skill or has already charged out, while under the effect of BKB, sinister gaze can interrupt it. 
3.  In the provided match, at 39:34, SB used BKB first then charges lich, lich interrupted charge with sinister gaze. 

EDIT: After some further testing, it seems like this bug is affecting skills that involve movement. I tested Blast off, icarus dive, skewer, and timber chain. These skills can be interrupted by sinister gaze while under the effect of BKB. I tried many other channeled skills, such as freezing field, death ward, black hole, dismember, multishot and tp. While under the effect of BKB, none of these skills can be interrupted. 

### Example Match ID (and possibly Timestamp)

7415466465 39:34

**Comment by Luojunya1991** [comment]
I believe this bug is quite significant. If people found out earlier, lich can be a hard counter to magnus, since sinister gaze can interrupt BKB skewer. It would have changed the course of TI results.

**Comment by TobyFlendersonFromHR** [comment]
>  It would have changed the course of TI results
not really

**Comment by TobyFlendersonFromHR** [comment]
>  It would have changed the course of TI results

not really, no

**Comment by Luojunya1991** [comment]
> > It would have changed the course of TI results
> 
> not really, no

Yes or no, that's just personal opinions. This is still a significant bug that needs to be fixed asap regardless

**Comment by Philaeux** [comment]
Illustration: https://www.youtube.com/watch?v=pZuSufeK2JI

**Comment by Luojunya1991** [comment]
> Illustration: https://www.youtube.com/watch?v=pZuSufeK2JI

Thank you for the illustration! As mentioned in the edit part, this bug also exists for other abilities that involve movement, such as blast off, skewer, Icarus dive and timber chain.

**Comment by Philaeux** [comment]
Yeah seems to be a Spell channeling stop going through

**Comment by Luojunya1991** [comment]
This is my first time submitting a bug, does anyone know a rough response time for bugs?

**Comment by Philaeux** [comment]
Can be today, can be never: Valve Time
